packages/one-trust-notices/src/config/config.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AWSConfig,
+  FileManagerConfig,
+  LoggerConfig,
+  S3Config,
+} from "pagopa-interop-commons";
+import { z } from "zod";
+import dotenv from "dotenv";
+
+export const OnetrustServiceConfig = FileManagerConfig.and(S3Config)
+  .and(LoggerConfig)
+  .and(AWSConfig)
```

Comment:
I would add FileManagerConfig so that you can run it locally using Minio instead of S3, and add the corresponding vars in the Env as in other services that use the file manager

```suggestion
  .and(AWSConfig)
  .and(FileManagerConfig)
```